<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CenterLineExtractView</class>
 <widget class="QWidget" name="CenterLineExtractView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>411</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Tube Model</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QmitkDataStorageComboBox" name="DataSelector"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QmitkPointListWidget" name="SeedWidget" native="true"/>
    </item>
    <item>
     <widget class="QPushButton" name="ApplyBtn">
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QmitkPointListView" name="EndPointList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>411</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="SelectEndPointBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>630</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QmitkDataStorageComboBox</class>
   <extends>QComboBox</extends>
   <header>QmitkDataStorageComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>QmitkPointListWidget</class>
   <extends>QWidget</extends>
   <header>QmitkPointListWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QmitkPointListView</class>
   <extends>QListView</extends>
   <header>QmitkPointListView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
